sdk.yaml

```diff
@@ -413,3 +413,12 @@ REST:
     .synopsis: |
       Convenience for customers who wish to use REST API functionality that is either not documented or is not included in the API for our client libraries.
       The REST client library provides a function to issue HTTP requests to the Ably endpoints with all the built in functionality of the library such as authentication, pagination, fallback hosts, MsgPack and JSON support etc.
+  Statistics:
+    .synopsis: |
+      Operations relating to Ably application statistics (a.k.a. 'stats').
+    Query:
+      .specification: [RSC6, RTC5]
+      .synopsis: |
+        Obtain statistics at 1 minute intervals, or aggregated up to the hour, day, or month.
+        Uses the `/stats` REST endpoint.
+        Results are provided aggregated across all channels in use in the application in the specified period and may be used to track usage against account quotas.
```

Comment:
Would it be useful to link out to more detailed documentation? So for example here, link to https://ably.com/docs/general/statistics?

Reply:
Good point. Done in https://github.com/ably/features/pull/16/commits/95cad85418693a70a52c3a702b2df62e69d60086.

Reply:
I'm going to resolve this conversation on @lmars' behalf in order to get this pull request merged.